### ISSUE TITLE
Update Gem dependencies

### ISSUE THREAD

**Issue by jeremylynch** [meilisearch/meilisearch-rails#354]
[Meilisearch 0.27.1](https://github.com/meilisearch/meilisearch-ruby/releases) adds support for the latest version of meilisearch (1.7.0), however this gem requires 0.26.0. Can we have the depdencies upgraded to allow Meilisearch 0.27.1? 

```
Because meilisearch-rails >= 0.10.2 depends on meilisearch ~> 0.26.0
  and Gemfile depends on meilisearch = 0.27.1,
  meilisearch-rails >= 0.10.2 cannot be used.
So, because Gemfile depends on meilisearch-rails = 0.12.0,
  version solving has failed.
```

**Comment by brunoocasali** [comment]
Hi @jeremylynch I just stumble upon your issue. 

We are using the ~> 0.26.0 so it should already allow you to update your meilisearch-ruby and meilisearch-rails version, no?

**Comment by ellnix** [comment]
> We are using the ~> 0.26.0 so it should already allow you to update your meilisearch-ruby and meilisearch-rails version, no?

`~>` only allows an update to the patch version, not the minor version.

From [the bundler documentation](https://bundler.io/guides/using_bundler_in_applications.html#adding-gems):
> ```ruby
> gem "rails", "~> 4.0.0" # which is same as gem "rails", ">= 4.0.0", "< 4.1.0" 
> ```

**Comment by ellnix** [comment]
> @ellnix I'm ok if you raise the dependency to `>= 0.26.0`

I think we should stick with `~>` and updating the version manually. If I change the requirement to `>=`, someone might want to use ms-rails 0.14 as we intended it and end up with a much later of ms-ruby that may no longer be compatible.

In any case, should I PR this and also make a `0.13.1` release?

**Comment by brunoocasali** [comment]
> In any case, should I PR this and also make a 0.13.1 release? (I feel like it would be better to update that on a minor version)

In my mind, we could always ignore the versions that are not compatible using `!= version` or just increase the starting point, let's say `meilisearch-ruby >= 0.30.0`. And we always ask the users to be on the latest on both sides :D 

But as you wish! I was trying to reduce the amount of work 😅

**Comment by jeremylynch** [comment]
Any update on this one?

**Comment by jeremylynch** [comment]
Any luck with this one?

**Comment by ellnix** [comment]
I believe this was implemented on #367
